libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -194,7 +194,7 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
                 row = location.start.line
                 column = location.start.column
                 locationString = row + 1 + ':' + column + ':'
-                fileName = Object.keys(lastCompilationResult.sources)[file]
+                fileName = Object.keys(lastCompilationResult.contracts)[file]
```

Comment:
is this a needed changes? doesn't seem related to the PR.

Reply:
Yes, i don't see any reason for this change.